library_conventions/index.adoc

```diff
@@ -0,0 +1,8 @@
+[#libraryconventions]
+== Library Conventions
+:imagesdir: library_conventions
+
+Here we collect conventions / guidelines to be used when designing libraries.
+
+include::symbols.adoc[]
+
```

Comment:
For some reason, when omitting this newline, the following API section is mangled together with the numeric list in symbols.adoc... Probably an asciidoctor bug.

Reply:
Moved the newline to the top level index.adoc. I'm not sure if it's a bug or if generally a newline is needed between include statements because one linebreak is typically ignored (like in many markup languages)...